jipdate.py

```diff
@@ -337,6 +341,16 @@ def create_default_config(config_file):
 # Extra comments added to each Jira issue (multiline is OK)
 comments:
         - "No updates since last week."
+
+# Header of the file (multiline is OK). It will be followed by JIRA_USERNAME
+header:
```

Comment:
I like this (patch) I even planned to do the same, but I think you could remove the old "header" defined at line 157 in jipdate.py. Agree?